app/javascript/dashboard/helper/editorHelper.js

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
Should we move these to @chatwoot/utils or should we move @chatwoot/utils to here and make this a mono repo? 

Have a consistent utils with documentation pattern would help in mobile app as well.

Reply:
Sounds good, I can move this to @chatwoot/utils for now. 

Making this a mono-repo will require some setup. We perhaps take it up later.